config/config.js

```diff
@@ -0,0 +1,9 @@
+const developmentConfig = require('./development-config');
+const testConfig = require('./test-config');
+
+if (process.env.NODE_ENV === 'test') {
```

Comment:
Directory structure should be as follows

- config/index.js
- config/development.js
- conifg/test.js
- config/production.js

All the common configs will be exported from index.js and overridable based on the NODE_ENV.